web3j/src/main/java/com/vmware/ethereum/service/SecureTokenApi.java

```diff
@@ -84,10 +98,40 @@ private SecurityToken deploy() {
                 initialSupply)
             .send();
 
-    securityToken.getTransactionReceipt().ifPresent(receipt -> log.info("Receipt: {}", receipt));
+    securityToken
+        .getTransactionReceipt()
+        .ifPresent(
+            receipt -> {
+              log.info("Receipt: {}", receipt);
+              contractAddress = receipt.getContractAddress();
+            });
     return securityToken;
   }
 
+  /** Transfer token for deferred polling. */
+  public void transferQueued() throws IOException, TransactionException {
```

Comment:
This method make a synchronous network call. It is invoked from WorkloadCommand.run() which is scheduled at fixed rate in open model. This would prevent you from injecting at desired arrival rate. Think about it, we'll discuss next week.